### DISCUSSION TITLE
Trojan:Script/Wacatac.B!ml

### DISCUSSION THREAD

**Discussion by DocAtPrompt** [vlang/v#17323] [Installation Issues]
by compiling vlang on my w11 machine defender worries about Trojan:Script/Wacatac.B!ml. Is the rep infected or is this an artifact?

**Comment by JalonSolov** [comment]
How did you install V?  The best way is via these instructions:  https://github.com/vlang/v#installing-v-from-source

V doesn't have any trojans/virii/whatever, unless wherever you got it from added something.

**Reply by DocAtPrompt** [reply]
I did it exaclty as described at that url
<img width="365" alt="Screenshot 2023-02-15 135740" src="https://user-images.githubusercontent.com/5602430/219033759-f3d327b6-d835-4946-9ed4-7f0ebdff2ed6.png">
<img width="371" alt="Screenshot 2023-02-15 135656" src="https://user-images.githubusercontent.com/5602430/219033789-d24328ca-fb70-4146-82de-ed7d24dd5485.png">

**Reply by DocAtPrompt** [reply]
no reproach, just asking

**Reply by JalonSolov** [reply]
Then it is simply a "false positive" from your virus checker.  Most of the virus companies allow you to submit a report about false positives, so they can make an exception.

**Reply by DocAtPrompt** [reply]
ok thank u, will do that...

**Reply by Wajinn** [reply]
When a person gets a false positive, they should report it (including where it came from) and upload the file.  Microsoft will usually give you feedback about it, which takes around a day.  It's helpful to say where you got the file from and that you think it's a false positive.  For Windows defender, here is the link.

https://www.microsoft.com/en-us/wdsi/filesubmission

Additionally, you can have an infected computer, and the original file causing the issue came from elsewhere.  A lot of people will be downloading from suspicious or odd sites, and then make the wrong assumptions about where an infection came from.  Wacatac.B!ml is not a new type of trojan, and has existed for many years.  !ml seems to indicate being a heuristic versus direct match, and can be triggered by lots of different files.  You should probably do a full scan of your computer.

**Comment by rakshasha-medhi** [comment]
> by compiling vlang on my w11 machine defender worries about Trojan:Script/Wacatac.B!ml. Is the rep infected or is this an artifact?

I'm getting this as well when trying to run the fibonacci.v example. 
I was following the step by step instructions to install V...so should be nothing's wrong in the process.
Flagged the file as false positive.

Interestingly, when I run hello_world.v it was okay.

**Reply by farteryhr** [reply]
reminds me of the old time when i write a prime sieve in certain way in C and hit compile, some antivirus software pops up and eats it :facepalm:

**Comment by vcma** [comment]
https://www.virustotal.com/gui/file/d902fbc8b7d6f3eafda4bb400db9546d881ff8ccfa749bcca6d98ff1fdd36182/detection

**Comment by Wajinn** [comment]
Just like with Microsoft (above), false positive reports can be submitted to GData and Ikarus. This includes submitting a false positive report to VirusTotal as well, for complaints about unresponsive or bad data from certain companies. The more people that contact them, the more likely or the sooner the problem or misunderstanding will be resolved.

```
GData: 

e-mail: support@gdata.pl
e-mail: sales@gdata.pl

https://gdata.pl/support/contact
https://gdata.pl/support/zglos-podejrzany-plik // to submit file
https://submit.gdatasoftware.com/privacy // to submit file

Ikarus:

e-mail: office@ikarus.at
e-mail: support@ikarus.at
```

VirusTotal:
[https://www.virustotal.com/gui/contact-us/support](
https://www.virustotal.com/gui/contact-us/support) // My site or file has been improperly flagged as harmful
